#!/usr/bin/env python

PACKAGE = "zerog_utilities"  # 

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("averaging_interval", double_t, 0, "Averaging Interval for Velocity Calculation", 0.0001, 0.01, 1.0)

exit(gen.generate(PACKAGE, "TfInfo", "TfInfo"))
